src/store/ChartState.js

```diff
@@ -358,15 +362,17 @@ class ChartState {
         }
     }
 
-    scrollChartToLeft = () => {
+    scrollChartToLeft = (leftTick, force) => {
```

Comment:
@shayanbinary We don't need to add an extra parameter to the function since the function is invoked with `force` as `true` when `leftTick` is also passed. so you can't check whenever the passed parameter (`leftTick`) has value, the animation shouldn't be run. I have added a suggestion on `if` statement line.

Reply:
@morteza-binary navigation widget is not the only component use `scrollChartToLeft`, Feed.js use it as well. mean when you click on `purchase`, `scrollChartToLeft` called with a `leftTick` which is should run ScrollAnimator, that's why I add another prop

Reply:
@shayanbinary Would you please let me know which line of `Feed.js` is using `scrollChartToLeft`?